src/utils/validator.ts

```diff
@@ -0,0 +1,5 @@
+export const validateId = (id: string) => {
+  if (!id.match("/^[a-f]{24}$/i")) {
+    throw new Error("Wrong ID format");
+  }
```

Comment:
@CeEv, may I ask what is this needed for?
If the user tries to enter roomName that does not exist he will be redirected to vue error page like the one below
![image](https://github.com/hpi-schul-cloud/tldraw-client/assets/27923014/b4130c74-7b4f-41a4-b6d7-725bb5eb4714)
Same thing if the user does not have access to the roomName he provided, just different text message

@wiaderwek 
If we do need this validation even before any of the above happens then throwing new Error will break entire app as there isn't any global error handling, it was suggested that we redirect to vue error page instead:

```
setErrorData(HttpStatusCode.NotFound, "tldraw.error.ws.4404");
redirectToErrorPage();
```